ex9_ch3.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: import math
+import math
+
```

Comment:
Is it possible to use a constant in this problem? For example, stating Area = L * W? Originally, I started out with a constant, but removed it because I wasn't sure how to get the user's input into the constant's equation.

Reply:
So a constant must be a single value, such as a string or number, that never changes. `area = L * W` is a _statement_ that changes with different values of `L` and `W`. Thus, this should not be a constant. I'll point out when you can use constants instead of normal variables.